src/containers/shared/components/Transaction/AMMBid/test/AMMBid.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { Simple } from '../Simple'
+import { createSimpleWrapperFactory, expectSimpleRowText } from '../../test'
+import bidMock from './mock_data/amm_bid.json'
+
+describe('AMM Bid Tests', () => {
+  const createWrapper = createSimpleWrapperFactory(Simple)
+
+  it('renders from transaction', () => {
+    const wrapper = createWrapper(bidMock)
+    expectSimpleRowText(wrapper, 'min_slot_price', 'LP 100')
+    expectSimpleRowText(wrapper, 'max_slot_price', 'LP 500')
+    expectSimpleRowText(
+      wrapper,
+      'account_id',
+      'rMEdVzU8mtEArzjrN9avm3kA675GX7ez8W',
+    )
+    expectSimpleRowText(
+      wrapper,
+      'auth_accounts',
+      'ra8uHq2Qme5j19TqvPzTE2nqT12Zc3xJmKrU6o2YguZi847RaiH2QGTkL4eZWZjbxZvk',
```

Comment:
Why is there no line break or space here?

Reply:
This is how it looks to the tester. In reality there's a new line in between auth accounts but when testing this is the string it returns. Not sure if it's because new lines arent registered here.